site/app/templates/grading/electronic/Status.twig

```diff
@@ -123,6 +123,24 @@
     }
 </script>
 <body onload="document.getElementById('defaultOpen').click();">
+<!-- If grader is limited access grader or student peer, Grading reminder should be displayed -->
+{% if core.getUser().getGroup() == 4 or core.getUser().getGroup() == 3%}
+<div class="content gradeable_message">
+    <h1>Your Responsibility as a Peer Grader!</h1>
```

Comment:
Should we give a slightly different message for limited access graders (at least not calling them peer graders)?

Reply:
@bmcutler What should the message be?